### DISCUSSION TITLE
GitHub student reverification issue

### DISCUSSION THREAD

**Discussion by Codeh4ck** [community/community#23304] [New to GitHub]
<p>Hello.</p>
<p>I’ve been asked by GitHub to reverify my student status. One of the methods I can use to verify is by showing a screenshot of my college’s portal with a timestamp, <a href="https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/#what-proof-do-i-need-to-submit-for-academic-verification" rel="nofollow noopener">according to this page</a>. I’ve sent a screenshot of my college’s portal with my user logged in and the timestamp is visible on my computer. However, it was rejected as “missing timestamp”.</p>
<p><strong>What sort of timestamp is required to pass the verification process?</strong></p>
<p>Thank you in advance.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/codeh4ck">@codeh4ck</a>,</p>
<p>To ensure that we’re able to provide the most accurate answer to your particular student status verification issue, we kindly ask that you <a href="https://support.github.com/contact" rel="nofollow noopener">reach out to our Support team for help</a>. They’ll be happy to give you detailed information about your application that we’re unable to share in a public forum.</p>
